.goreleaser.yml

```diff
@@ -26,9 +26,10 @@ archives:
       - goos: 'windows'
         format: 'zip'
 
-    wrap_in_directory: true
+    wrap_in_directory: false
```

Comment:
Let this be true only as we have already submitted the plugin with path.

Reply:
We'd need to re-submit it as likely the krew CI looks for LICENSE file in the root

Reply:
I didn't look for docs which can explicitly specify the relative path of the LICENSE & README.md(these files should also be shipped with the binary).

Reply:
Re-submit as in? Raise another PR?

Reply:
Yes, but we can keep this true and update the paths for krew-CI spec too, e.g. [plugins/resource-snapshot.yaml](https://github.com/kubernetes-sigs/krew-index/blob/master/plugins/resource-snapshot.yaml#L28)

Reply:
> Re-submit as in? Raise another PR?

Yes, we'd have to manually submit once for 0.3.0 if I change the relative path of the binary, later that can be automated via their GitHub action(sooo coool) 😄

Reply:
Yes. I was looking at their publisher bot. Maybe I will try to set that up. Please do whatever suits best for this PR.

Reply:
I've wrapped the binaries inside the directory, I'll update the configuration for krew by checking with my own krew index.

> Yes. I was looking at their publisher bot.
Yes, we'd do that from 0.4.0 releases.